policies/communication_channels.md

```diff
@@ -0,0 +1,63 @@
+# Communication channels in the Astropy Project
+
+## Preface
+
+Communication in the Astropy Project is governed by the [Community
+Code of Conduct](https://www.astropy.org/code_of_conduct.html), which
+takes precedence over anything written here. This goal of this policy is
+to write down the technical details of how email lists, Slack
+channels, forums etc. are set up.
+
+While we aim to be an inclusive and welcoming community, we
+recognize that not all users want to be involved in the technical
+details of building and running a library like Astropy. Therefore, some
+communication channels are more geared towards users and others
+to developers. Also, in a project this size, certain issues that
+arise require privacy (e.g. personal or financial information is
+legally protected in many countries). Thus, there are communication
+channels geared towards or exclusively for developers, maintainers, or
+members of certain committees/roles.
+
+
+## Email communication geared towards project members
+
+Mailing lists are maintained as Google groups or through other list
+servers.  Two examples of such lists are astropy@python.org, geared
+towards users, and [astropy-dev](http://groups.google.com/group/astropy-dev), which is intended for more
+technical developer discussions.  Unless specifically restricted,
+anyone worldwide can subscribe to these lists, and previous posts are
+publicly available. List membership is required to post messages to
+prevent spam.  In order to be as open as possible, none of these lists
+shall be moderated, nor will posts by group members be otherwise
```

Comment:
What does "not moderated" mean here, things in violation of the CoC or spam etc will be removed from the archives wont they?

Reply:
If you retroactively remove a message from google groups, i.e. not have everything approved, then all it's doing is removing it from the archive (and if you get to it in time preventing it going out in digest emails).